stdlib/tests/collections/smt.rs

```diff
@@ -274,13 +274,13 @@ fn prepare_insert_or_set(
     (initial_stack, expected_output, store, advice_map)
 }
 
-fn build_advice_inputs(smt: &Smt) -> (MerkleStore, Vec<([u8; 32], Vec<Felt>)>) {
+fn build_advice_inputs(smt: &Smt) -> (MerkleStore, Vec<(RpoDigest, Vec<Felt>)>) {
     let store = MerkleStore::from(smt);
     let advice_map = smt
         .leaves()
         .map(|(_, leaf)| {
             let leaf_hash = leaf.hash();
-            (leaf_hash.as_bytes(), leaf.to_elements())
+            (RpoDigest::try_from(leaf_hash.as_bytes()).unwrap(), leaf.to_elements())
```

Comment:
Isn't `leaf_hash` already an `RpoDigest`? If so, we don't need to do this conversion to/from bytes here.